tests/python/pants_test/engine/test_build_files.py

```diff
@@ -39,79 +40,91 @@ def test_empty(self):
 
 
 class AddressesFromAddressFamiliesTest(unittest.TestCase):
+
+  def setUp(self):
+    self._default_address_mapper = AddressMapper(JsonParser(TestTable()))
+    self._default_snapshot = Snapshot(DirectoryDigest('xx', 2),
```

Comment:
Rather than making these properties, prefer methods: that makes them easier to template later: ie `self._snapshot()` vs `self._snapshot(with_more_stuff=True)`.

Reply:
Done in 5801d95a1! Let me know if these need any explanatory comments or docstrings.

Reply:
Github isn't letting me reply to your other comment, but I also made `_MappedSpecs` private in 11bd1c609.